jobs/config.go

```diff
@@ -24,6 +24,7 @@ type Config struct {
 
 	// service discovery
 	Port              int           `mapstructure:"port"`
+	InitialStatus     string        `mapstructure:"initialStatus"`
```

Comment:
Let's call it `initial_status` (snake case), I don't believe we use lower camel case anywhere else. In Go we'll continue to use `InitialStatus`.

Reply:
Ah yes, you're right.